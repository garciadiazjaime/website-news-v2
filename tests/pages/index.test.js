import React from 'react'
import renderer from 'react-test-renderer'
// import placesSample from './placesSample.json'

import HomePage from '../../pages/index'

describe('index.js', () => {
  it('renders properly', () => {
    const props = {
      url: {
        pathname: ''
      },
      places: []
    };
    const component = renderer.create(<HomePage {...props} />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
