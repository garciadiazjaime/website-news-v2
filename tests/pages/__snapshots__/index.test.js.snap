// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.js renders properly 1`] = `
<div
  className="jsx-4167571301"
>
  <div
    className="jsx-1288317031"
  >
    <header
      className="jsx-1288317031"
    >
      <h1
        className="jsx-1288317031"
      >
        Restaurantes, Bares y Caf√©s en Playas de Tijuana
      </h1>
    </header>
  </div>
</div>
`;
